examples/produces_consumes/produces_consumes.bal

```diff
@@ -6,26 +6,26 @@ endpoint http:Listener infoServiceEP {
     port:9092
 };
 
-//Consumes and produces annotations that contain MIME types as an array of strings.
+// Consumes and produces annotations that contain MIME types as an array of strings.
 service<http:Service> infoService bind infoServiceEP {
```

Comment:
We can say ```ballerina service<http:Service> infoService bind {port:9092} ``` so we can get rid of infoServiceEP.

Reply:
Please check other samples 
- https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/http_caching_client/http_caching_client.bal
- https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/http_circuit_breaker/http_circuit_breaker.bal